repository/constants.go

```diff
@@ -10,8 +10,8 @@ var (
 	CMD   = filepath.Base(os.Args[0])
 	Flags = []Flag{
 		{"e", "Display the word in a sentence"},
+		{"ss", "Display a short and sweet version of the definition"},
 		{"t", "Display only thesaurus results"},
```

Comment:
"Display only thesaurus results" makes it sound like thesaurus results are included in a query for definitions...